File: src/main/java/tutorial/ref/java/employee/Employee.java
package tutorial.ref.java.employee;

import tutorial.ref.java.address.Address;
import tutorial.ref.java.person.Person;

public class Employee {
    private Person person;
    private Address address;
    private String emp_id;
    private String salary;

    @Override
    public String toString() {
        return "Employee{" +
                "person=" + person +
                ", address=" + address +
                ", emp_id='" + emp_id + '\'' +
                ", salary='" + salary + '\'' +
                '}';
    }

    public Person getPerson() {
        return person;
    }

    public void setPerson(Person person) {
        this.person = person;
    }

    public Address getAddress() {
        return address;
    }

    public void setAddress(Address address) {
        this.address = address;
    }

    public String getEmp_id() {
        return emp_id;
    }

    public void setEmp_id(String emp_id) {
        this.emp_id = emp_id;
    }

    public String getSalary() {
        return salary;
    }

    public void setSalary(String salary) {
        this.salary = salary;
    }

    public Employee() {
    }

    public Employee(Person person, Address address, String emp_id, String salary) {
        this.person = person;
        this.address = address;
        this.emp_id = emp_id;
        this.salary = salary;
    }
}

File: src/main/java/tutorial/ref/kotlin/employee/Employee.kt
package tutorial.ref.kotlin.employee

import tutorial.ref.kotlin.address.Address
import tutorial.ref.kotlin.person.Person


class Employee {
    var emp_id:String? = null
    var salary:String? = null
    var person: Person? = null
    var address: Address?=null

    constructor()
    constructor(emp_id: String?, salary: String?, person: Person?, address: Address?) {
        this.emp_id = emp_id
        this.salary = salary
        this.person = person
        this.address = address
    }

    override fun toString(): String {
        return "Employee(emp_id=$emp_id, salary=$salary, person=$person, address=$address)"
    }
}